cmd/swarm/flags.go

```diff
@@ -269,4 +265,8 @@ var (
 		Name:  "block-profile",
 		Usage: "Enable pprof block profile",
 	}
+	SwarmPublisher = cli.StringFlag{
+		Name:  "publisher",
+		Usage: "Manually specify Publisher",
```

Comment:
let's be a little more descriptive here; something like `Manually specify content recovery publisher for a hash`